test/extensions/transport_sockets/alts/tsi_handshaker_test.cc

```diff
@@ -176,7 +181,8 @@ class AltsTsiHandshakerTest : public Test {
       server_ = server_builder.BuildAndStart();
       EXPECT_THAT(server_, NotNull());
       EXPECT_NE(listening_port, -1);
-      server_address_ = absl::StrCat("[::1]:", listening_port);
+      server_address_ =
+          absl::StrCat(Network::Test::getLoopbackAddressUrlString(version_), ":", listening_port);
```

Comment:
Looks like there is build error.  `version_` is in test base `HttpIntegrationTest` or base_integration_test.h

Reply:
Right, most of the tests are not inheriting `HttpIntegrationTest`.